thirdparty/catch2/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'cpp'
+apply plugin: 'visual-studio'
+apply plugin: 'edu.wpi.first.NativeUtils'
+
+ext {
+    nativeName = 'Catch2'
+}
+
+apply from: "${rootDir}/shared/config.gradle"
+
+model {
+    components {
+        "${nativeName}"(NativeLibrarySpec) {
+            sources.cpp {
+                source {
+                    srcDirs "src/main/native/cpp"
+                    include '*.cpp'
```

Comment:
This does not pull in .cpp files from subdirectories.  It needs to be something like `'**/*.cpp'`.